Wikipedia/Code/WMFImageController.swift

```diff
@@ -411,54 +390,39 @@ extension WMFImageController {
      - returns: `AnyPromise` which resolves to `WMFImageDownload`.
      */
     @objc public func fetchImageWithURL(url: NSURL?) -> AnyPromise {
-        return AnyPromise(bound: fetchImageWithURL(url))
-    }
-    
-    /**
-     Objective-C-compatible variant of fetchImageWithURL(url:options:) returning an `AnyPromise`.
-     
-     - returns: `AnyPromise` which resolves to `WMFImageDownload`.
-     */
-    @objc public func fetchImageWithURL(url: NSURL?, options: SDWebImageOptions) -> AnyPromise {
-        return AnyPromise(bound: fetchImageWithURL(url, options: options))
+        return AnyPromise(bound: Promise<WMFImageDownload> { fulfill, reject in
```

Comment:
@coreyfloyd wanted to verify with you and monte that this is the pattern we want to use before updating all the obj-c to the new normal `fetchImageWithURL`